std/typecons.d

```diff
@@ -1921,19 +1921,32 @@ Returns:
         return _value;
     }
 
-///
-unittest
-{
-    import std.exception: assertThrown, assertNotThrown;
-
-    Nullable!int ni;
-    //`get` is implicitly called. Will throw
-    //an AssertError in non-release mode
-    assertThrown!Throwable(ni == 0);
+/**
+Forwads $(D T) opEquals to handle the null state.
 
-    ni = 0;
-    assertNotThrown!Throwable(ni == 0);
-}
+Returns:
+    $(D true) if two nullables are in null state,
+    $(D false) if two nullables have a different null state,
+    the values comparison result otherwise.
+*/
+    bool opEquals(R)(R rhs)
```

Comment:
Please make this method const so it works for `immutable` and `const` `Nullables`.